addons/l10n_ua/data/account_chart_template.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+
+    <record id="l10n_ua_psbo_chart_template" model="account.chart.template">
+        <field name="name">План рахунків ПСБО</field>
+        <field name="cash_account_code_prefix">301</field>
+        <field name="bank_account_code_prefix">311</field>
+        <field name="transfer_account_code_prefix">317</field>
```

Comment:
333

Reply:
Sorry, 333 for what?

Reply:
According to the chart of accounts in this document ( https://www.pwc.com/ua/en/solutions/assets/doing_business_and_investing_in_ukraine_2010.pdf ) 317 would be under bank accounts an 333 under other accounts.

Reply:
yes, but we have explicit government explanation to use for that purpose accounts under 33, they translated as 'Money on transferring' (smth like this), because we not have them right now etc.. long story,
 it's not so important as in Ukraine in reality no one use it

Reply:
Yep, 333 is correct for transfer account.
According to law https://zakon.rada.gov.ua/laws/show/z0893-99